pkg/util/lifted/pubkeypin/pubkeypin.go

```diff
@@ -56,18 +57,18 @@ func (s *Set) Allow(pubKeyHashes ...string) error {
 	for _, pubKeyHash := range pubKeyHashes {
 		parts := strings.Split(pubKeyHash, ":")
 		if len(parts) != 2 {
-			return fmt.Errorf("invalid hash, expected \"format:hex-value\". "+
+			return errors.Errorf("invalid hash, expected \"format:hex-value\". "+
```

Comment:
Please don't use `errors` as the package has been archived. `fmt` would be fine.

Reply:
> Please don't use `errors` as the package has been archived. `fmt` would be fine.

ok